### PR TITLE
JAMES-2869 Update website's Java version

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: src/homepage/index.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h2>About</h2>
               <div class="about-table">
                 <h3 class="tb-h3">James from a technical point of view</h3>
                 <b>Complete portability</b> (100% pure Java)<br/>
-                <b>Running on Java 6 onwards</b> and on the <b>JVM</b>. Some optional components might require Java 8.<br/><br/>
+                <b>Built with Java 8</b> and running on the <b>JVM</b>.<br/><br/>
```
Comment (chibenwa) [inline src/homepage/index.html]: Don't we require java 8 for runnig as well? (`<target.jdk>1.8</target.jdk>`)

Comment (blackheaven) [reply src/homepage/index.html]: s/JVM/JRE 1.8/ ?

**Comment by blackheaven** [comment]
```
[INFO] Apache James :: Server :: MailRepository :: Cassandra FAILURE [03:59 min]
```

test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**